webpack/assets/javascripts/react_app/actions/NotificationActions.js

```diff
@@ -33,7 +33,12 @@ export default {
       expand: group
     });
   },
-  markAsRead(url) {
-    API.markNotificationAsRead(url);
+  markAsRead(id, group) {
+    API.markNotificationAsRead(`/notification_recipients/${id}`);
```

Comment:
should we store the API here? I think it makes more sense to include the URL in the markNotificationAsRead function and only pass the ID ?